### PR TITLE
Feature/osw

### PR DESCRIPTION
**Pull request by hroest**
- add documentation
- change column order in TSV
- bugfix if RT scores are missing

### CODE REVIEW

**Comment by hroest** [comment]
@grosenberger can you have a look at the documentation of OSW files and maybe comment on them if they are accurate?

File: src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h [hunk 1]
```diff
@@ -47,11 +47,70 @@ namespace OpenMS
...
+        <tr> <th BGCOLOR="#EBEBEB" colspan=3>RUN</th> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">ID</td> <td>INT</td> <td> Primary Key (run id)</td> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">FILENAME</td> <td>TEXT</td> <td> Original filename associated with the run </td> </tr>
+      </table>
+
+      <table>
+        <tr> <th BGCOLOR="#EBEBEB" colspan=3>FEATURE</th> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">ID</td> <td>INT</td> <td> Primary Key (feature id)</td> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">RUN_ID</td> <td>INT</td> <td> Foreign Key (RUN.ID)</td> </tr>
+        <tr> <td BGCOLOR="#EBEBEB">PRECURSOR_ID</td> <td>INT</td> <td> Foreign Key (TransitionPQPFile PRECURSOR.ID) </td> </tr>
```
Comment (hroest) [inline src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h]: @grosenberger is this correct, the idea of this is that the PRECURSOR_ID relates to the TransitionPQPFile PRECURSOR.ID column, correct?

Comment (grosenberger) [reply src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h]: Yes, that is correct.

**Comment by cbielow** [comment]
+1

**Pull request merged**